src/main/java/ua/danit/final_project/services/websocket/WebSocketService.java

```diff
@@ -32,4 +35,14 @@ public void updateTask(TaskDto taskDto) throws JsonProcessingException {
   public void updateComment(ShiftCommentDto shiftCommentDto) throws JsonProcessingException {
     template.convertAndSend(EVENTS_WORKSHIFT_COMMENT, mapper.writeValueAsString(shiftCommentDto));
   }
+
+  public void deleteComment(Long id) throws JsonProcessingException {
+    template.convertAndSend(EVENTS_WORKSHIFT_COMMENT_REMOVAL, new RemovalResponse(id));
+  }
+
+  @AllArgsConstructor
+  @Data
+  private class RemovalResponse {
+    private final Long id;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "id" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=169&issues=AWbzfYJesSodgw_KO8wK&open=AWbzfYJesSodgw_KO8wK)

[ISSUE_KEY]: AWbzfYJesSodgw_KO8wK